src/helpers.ts

```diff
@@ -54,7 +115,9 @@ export async function getCorrespondingPackageFromModuleId(
         pkgJson = await getPackageJson(folder);
     }
 
-    if (pkgJson !== null) {
+    // some packages don't have names and act just as loader proxy - but we need to find
+    // the root module so we only resolve if we have the package, a name and a version
+    if (pkgJson !== null && pkgJson.version && pkgJson.name) {
```

Comment:
Discovered this issue during testing where importing `@mui/base/Button` resolved in the components `package.json` which doesn't contain a name and version.